splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/UpgradeScriptToCreateNewInvalidateStoredStatementsProcedure.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2012 - 2020 Splice Machine, Inc.
+ *
+ * This file is part of Splice Machine.
+ * Splice Machine is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3, or (at your option) any later version.
+ * Splice Machine is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License along with Splice Machine.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.splicemachine.derby.impl.sql.catalog.upgrade;
+
+import com.splicemachine.db.iapi.error.StandardException;
+import com.splicemachine.db.iapi.store.access.TransactionController;
+import com.splicemachine.derby.impl.sql.catalog.SpliceDataDictionary;
+import com.splicemachine.utils.SpliceLogUtils;
+
+/**
+ * Created by msirek on 11/7/20.
+ */
+public class UpgradeScriptToCreateNewInvalidateStoredStatementsProcedure extends UpgradeScriptBase {
+    public UpgradeScriptToCreateNewInvalidateStoredStatementsProcedure(SpliceDataDictionary sdd, TransactionController tc) {
+        super(sdd, tc);
+    }
+
+    @Override
+    protected void upgradeSystemTables() throws StandardException {
+        sdd.replaceInvalidateStoredStatementsAndDependentProcedures(tc);
+
+        SpliceLogUtils.info(LOG, "Catalog upgraded: created new version of SYSCS_UTIL.SYSCS_INVALIDATE_STORED_STATEMENTS.");
+
+    }
+}
```

Comment:
This is not needed. If software version is higher than data dictionary catalog version, createOrUpdateAllSystemProcedures() recreates all system procedures.

Reply:
Thanks @jyuanca, I removed the upgrade script.